src/components/TextInputAutoGrow.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import {getAutoGrowTextInputStyle, getHiddenElementOutsideOfWindow} from '../styles/styles';
```

Comment:
This should be `getHiddenElementOutsideOfWindowStyle()` to match other style helper methods